readme.md

```diff
@@ -94,6 +94,14 @@ export default {
     Page,
     Post
   },
+  // function(req, key) to change the redis cache key generated. Uses browser request and initial cache key as arguments and must return a string which replaces the initial cache key.
+  cacheKeyHandler: (request, cacheKey) => {
+    let newKey = cacheKey
+    if (request.headers['connection-speed'] > 'fast') {
+      newKey = cacheKey + 'fastuser'
+    }
+    return newKey
+  },
```

Comment:
This will apply to all local caches, we default to in memory cache if Redis is unavailable. https://www.npmjs.com/package/cache-manager.

I wonder whether we need to make sure the user doesn't modify `request` or `cacheKey` that we pass into the `cacheKeyHandler` function? Worth looking at that method to clone the data rather than pass a reference to it.

Reply:
I don't understand the memory cache bit ☹️ 

```_.clonedeep(headers)``` should clone it properly right?

For the cacheKey, [research suggests](https://www.htmlgoodies.com/html5/javascript/passing-javascript-function-arguments-by-reference.html) that a string is immutable and any attempt to change it creates a copy rather than modify the original